index.html

```diff
@@ -123,7 +123,7 @@
         <p>A deck which contains only Standard cards is called a <i>Standard deck</i>.</p>
         <h3>
           Banlist
-          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of January 2017."></a>
+          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of April 2017."></a>
```

Comment:
That's a good idea! I was also toying with linking the announcements in which they were banned.